File: python/DPR.py
class DPR:
    def __init__(self, id=0, name="", namaBidang="", partai=""):
        self.id = id
        self.name = name
        self.namaBidang = namaBidang
        self.partai = partai

    def get_Id(self):
        return self.id

    def set_Id(self, id):
        self.id = id

    def get_Name(self):
        return self.name

    def set_Name(self, name):
        self.name = name

    def get_namaBidang(self):
        return self.namaBidang

    def set_namaBidang(self, namaBidang):
        self.namaBidang = namaBidang

    def get_Partai(self):
        return self.partai

    def set_Partai(self, partai):
        self.partai = partai

File: java/DPR.java
public class DPR {
    private int id;
    private String name;
    private String namaBidang;
    private String partai;

    public DPR() {
        this.id = 0;
        this.name = "";
        this.namaBidang = "";
        this.partai = "";
    }

    public DPR(int id, String name, String namaBidang, String partai) {
        this.id = id;
        this.name = name;
        this.namaBidang = namaBidang;
        this.partai = partai;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getNamaBidang() {
        return namaBidang;
    }

    public void setNamaBidang(String namaBidang) {
        this.namaBidang = namaBidang;
    }

    public String getPartai() {
        return partai;
    }

    public void setPartai(String partai) {
        this.partai = partai;
    }
}
